package.json

```diff
@@ -17,7 +17,7 @@
     "express": "^4.16.3",
     "mongo": "^0.1.0",
     "mongodb": "^3.1.0",
-    "mongoose": "^5.1.7",
+    "mongoose": "^5.4.17",
```

Comment:
any specific reason for updating mongoose?

Reply:
I did not intend to do it.
During `npm install`, I got a prompt to execute `npm audit fix`. Should I revert those changes?

Reply:
Okay. I'll undo the changes. But, should I close this PR as it was already assigned?

Reply:
i don't think, let it be there and do comment on the issue that you have fixed it. Am not sure if @imskr already working on it.

PLS Don't repeat this as we don't want multiple people to work on the same issue.

Reply:
Sure, I'll keep that in mind.